src/Common.Authentication/Models/AzureProfile.cs

```diff
@@ -87,6 +140,14 @@ public AzureProfile(string path)
             LoadDefaultEnvironments();
         }
 
+        private void Initialize()
+        {
+            Environments = new Dictionary<string, AzureEnvironment>(StringComparer.InvariantCultureIgnoreCase);
+            Subscriptions = new Dictionary<Guid, AzureSubscription>();
+            Accounts = new Dictionary<string, AzureAccount>(StringComparer.InvariantCultureIgnoreCase);
+            CurrentContext = new AzureContext();
```

Comment:
This should be initialized to use DefaultSubscription and it's related info. Something like this
```csharp
// Assuming environments and accounts collections are loaded
CurrentContext.Subscritpion = DefaultSubscription;
CurrentContext.Environment = Environments[DefaultSubscription.Environment]
CurrentContext.Account = Accounts[DefaultSubscription.Account]
```